lambda_code/takeover/takeover.py

```diff
@@ -409,7 +409,7 @@ def lambda_handler(event, context):  # pylint:disable=unused-argument
 
                 takeover_domains.append(finding["Domain"])
 
-            elif ".elasticbeanstalk.com" in finding["Takeover"]:
+            elif finding["Takeover"].endswith(".elasticbeanstalk.com"):
```

Comment:
I suspect it's possible that the finding can include a period at the end, i.e. `.elasticbeanstalk.com.` - so if you want to avoid the current syntax, it would be important to cater for this possibility (or convince ourselves that this is not the case)

Reply:
AFAIK for generated domains AWS would not add that period at the end.

But we can just use a regex like for other domains and just include both cases.

Reply:
Added regex for elasticbeanstalk.